orchestra/templates/orchestra/index.html

```diff
@@ -93,12 +93,12 @@
 <script src="{% static "orchestra/project_management/js/projectVis.js" %}" type="text/javascript"></script>
 
 {% for js in javascript_includes %}
-    <script src="{{ js }}" type="text/javascript"></script>
+  {{ js | safe }}
```

Comment:
oh i see---should we have a comment here saying to check out `orchestra/views.py` for why we lazily use `{% static`